modules/apps/commerce/commerce-product-content-web/src/main/resources/META-INF/resources/product_publisher/render/list/entry/view.jsp

```diff
@@ -101,7 +103,7 @@ boolean hasMultipleCPSkus = cpContentHelper.hasMultipleCPSkus(cpCatalogEntry);
 
 			<div>
 				<c:choose>
-					<c:when test="<%= !hasMultipleCPSkus && (cpSku != null) %>">
+					<c:when test="<%= (!hasMultipleCPSkus && (cpSku != null)) && !hasOptions%>">
```

Comment:
Checking for options in general matches logic in item/render/view.jsp so I opted for that versus checking for more limited options (e.g. required only). This may be something we want to tweak though depending on user experience.